src/modules/esl-utils/dom/element.ts

```diff
@@ -0,0 +1,41 @@
+import {getWindow} from './window';
+
+/**
+ * Get an object containing the values of all CSS properties of an element.
+ * @param element - element for which to get the computed style
+ * */
+export function getComputedStyle(element: Element): CSSStyleDeclaration {
+  return getWindow(element).getComputedStyle(element);
```

Comment:
Do we know why do we need to make sure of the window object here?
It looks like an electron environment support shim basically or some super strange situation with iframes from the same domain.

Reply:
Got rid of this method.